ts/src/pro/okx.ts

```diff
@@ -201,7 +207,7 @@ export default class okx extends okxRest {
         }
         await this.loadMarkets ();
         symbols = this.marketSymbols (symbols);
-        const channel = 'trades';
+        const channel = this.safeString (params, 'channel', 'trades');
```

Comment:
we should always use  [ channel, params ] = this.handleOptionAndParams (params, 'watchTrades', 'channel', 'trades'); in these cases